website/docs/r/cluster.html.markdown

```diff
@@ -224,14 +224,19 @@ To prevent this a lifecycle customization should be used, i.e.:
 `lifecycle {
   ignore_changes = [provider_instance_size_name]
 }`
+But in order to explicitly change `provider_instance_size_name` comment the `lifecycle` block and run `terraform apply`. Please ensure to uncomment it to prevent any accidental changes.
```

Comment:
FYI for next time - normally you want to ensure you are just submitting changes for the specific PR.  So in this case it would be best to have a branch for the doc changes and submit that as a PR and then a branch for the actual example.  This is okay for this time but in the future try to keep each branch/PR to a specific ticket/assignment.  Also always be sure the branches are off master unless there's a special case.   Let me know if you have any questions.

Reply:
Sure will do that